rca/programmes/migrations/0089_remove_programmepageglobalfieldssettings_key_details_book_or_view_all_open_days_link_title_and_more.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 4.2.9 on 2024-02-29 07:14
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("programmes", "0088_django_40_upgrade"),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name="programmepageglobalfieldssettings",
+            name="key_details_book_or_view_all_open_days_link_title",
+        ),
+        migrations.AddField(
+            model_name="programmepage",
+            name="book_or_view_all_open_days_link_title",
+            field=models.CharField(
+                blank=True,
+                default="Book or view all open days",
+                max_length=255,
+                verbose_name="Book open days title",
+            ),
+        ),
+    ]
```

Comment:
**Suggestion**

Rename the migration file to something simpler.

Reply:
Thanks, done in 37712d16